src/test/java/pages/ProductsPage.java

```diff
@@ -0,0 +1,61 @@
+package pages;
+
+import helpers.ConfigurationManager;
+import io.qameta.allure.Step;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class ProductsPage extends AbstractPage {
+
+    protected final String PAGE_URL = ConfigurationManager.getProperty("base.url") + ConfigurationManager.getProperty("product.page.url");
```

Comment:
product.page.url - not a property. It relates to Page implementation. Please remove